host/integration_test.go

```diff
@@ -1603,7 +1603,7 @@ func (s *integrationSuite) TestContinueAsNewWorkflow() {
 	for i := 0; i < 10; i++ {
 		err := poller.pollAndProcessDecisionTask(false, false)
 		s.logger.Infof("pollAndProcessDecisionTask: %v", err)
-		s.Nil(err, i)
+		s.Nil(err, string(i))
```

Comment:
this is most likely not what you want, string(3) != "3".

Reply:
Thanks for catching this!  Fix in https://github.com/uber/cadence/pull/387